tests/integration/ganache/test_consume_flow.py

```diff
@@ -28,6 +28,7 @@ def test_consume_flow(
     consumer_wallet: Wallet,
     data_nft: DataNFT,
     file1: FilesType,
+    arweave_file: FilesType,
```

Comment:
instead of adding this parameter, I would suggest parameterising the contents of file1 using pytest. i.e. it would run a test where file1 is regular, otherwise it is an arweave.

Reply:
I think the downside of parameterizing is that it increases CI run times, but I'll assume you thought of this, and proceed with the requested changes.